pypesto/objective/amici_calculator.py

```diff
@@ -71,6 +71,11 @@ def __call__(self,
             requested).
         """
         # set order in solver
+        if sensi_orders:
+            sensi_order = max(sensi_orders)
+        else:
+            sensi_order = 0
+
```

Comment:
If sensi_order is only used in the amici_solver.setSensitivityOrder, would perhaps do it like in the other changes with 
```python
if 2 in sense_orders:
...
```

Reply:
Amici will always only compute lower sensitivity orders, as they come more or less for free. For other objectives that might not be the case, hence the distinction.